observations/atmosphere/aircraft.yaml.j2

```diff
@@ -158,199 +564,412 @@
+    - variable: PreQC/windNorthward
+      is_in: 9-15
+    action:
+      name: reject
+    defer to post: true
+
+#  # errormod
+#  - filter: Perform Action
+#    filter variables:
+#    - name: windEastward
+#    action:
+#      name: inflate error
+#      inflation variable:
+#        name: ObsFunction/ObsErrorFactorConventional
+#        options:
+#          test QCflag: PreQC
+#          inflate variables: [windEastward]
+#          pressure: MetaData/pressure
+#          distance threshold: 60000.0
+#    defer to post: true
+#
+#  - filter: Perform Action
+#    filter variables:
+#    - name: windNorthward
+#    action:
+#      name: inflate error
+#      inflation variable:
+#        name: ObsFunction/ObsErrorFactorConventional
+#        options:
+#          test QCflag: PreQC
+#          inflate variables: [windNorthward]
+#          pressure: MetaData/pressure
+#          distance threshold: 60000.0
+#    defer to post: true
+
+  # Error inflation when QC is 3 or 7
+  - filter: Perform Action
+    filter variables:
+    - name: windEastward
+    - name: windNorthward
+    action:
+      name: inflate error
+      inflation factor: 1.2
+    where:
+    - variable: PreQC/windNorthward
+      is_in: 3,7 
+
+  # Error inflation when observation pressure < 50 hPa (read_prepbufr.f90)
+  - filter: Perform Action
+    filter variables:
+    - name: windEastward
+    - name: windNorthward
+    action:
+      name: inflate error
+      inflation factor: 1.2
+    where:
+    - variable: MetaData/pressure
+      maxvalue: 5000.0
+
+  # Make sure wind data where QC is 1-7 is accepted. 
+  - filter: Perform Action
     filter variables:
     - name: windEastward
     - name: windNorthward
-    test variables:
-    - name: ObsFunction/WindDirAngleDiff
+    where:
+    - variable: PreQC/windNorthward
+      is_in: 1-7
+    action:
+      name: accept
+
+  # Error inflation based on pressure check
+  - filter: Perform Action
+    filter variables:
+    - name: windEastward
+    action:
+      name: inflate error
+      inflation variable:
+        name: ObsFunction/ObsErrorFactorPressureCheck
+        options:
+          variable: windEastward
+          inflation factor: 4.0
+          surface_obs: false
+          #adjusted_error_name: GsiAdjustObsError
+          test_obserr: GsiAdjustObsError
+          #geovar_sfc_geomz: surface_geometric_height
+          geovar_sfc_geomz: height_above_mean_sea_level_at_surface
+          #geovar_sfc_geomz: geopotential_height_at_surface
+    defer to post: true
+
+  # Error inflation based on pressure check
+  - filter: Perform Action
+    filter variables:
+    - name: windNorthward
+    action:
+      name: inflate error
+      inflation variable:
+        name: ObsFunction/ObsErrorFactorPressureCheck
+        options:
+          variable: windNorthward
+          inflation factor: 4.0
+          surface_obs: false
+          #adjusted_error_name: GsiAdjustObsError
+          test_obserr: GsiAdjustObsError
```

Comment:
Why do we use test_obserr here?

Reply:
This is in order to use GsiAdjustObsError, though this may have to be removed in the future as this means the file has a GSI dependency.